.gitignore

```diff
@@ -16,3 +16,4 @@ build/*
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+*.DS_Store
```

Comment:
This was annoying me, so I plopped this in here. Let me know if this should be in a separate PR.